war/src/main/less/modules/buttons.less

```diff
@@ -1,77 +1,77 @@
 @import '../abstracts/colors.less';
 @import '../abstracts/font.less';
 
-// Button variant mixins
+// Button: variant mixins
 
 .button-danger() {
-  color: @white;
-  background-color: @danger;
-  border-color: @danger;
+  color: var(--btn-text-color);
+  background-color: var(--danger);
+  border-color: var(--danger);
 
   &:hover,
   &:focus {
-    color: @white;
-    background-color: @danger-hover;
-    border-color: @danger-hover;
+    color: var(--btn-text-color);
+    background-color: var(--danger-hover);
+    border-color: var(--danger-hover);
   }
 
   &:active {
-    color: @white;
-    background-color: @danger-active;
-    border-color: @danger-active;
+    color: var(--btn-text-color);
+    background-color: var(--danger-active);
+    border-color: var(--danger-active);
   }
 }
 
 .button-primary() {
-  color: @white;
-  background-color: @btn-primary-bg;
-  border-color: @btn-primary-bg;
+  color: var(--btn-text-color);
+  background-color: var(--btn-primary-bg);
+  border-color: var(--btn-primary-bg);
```

Comment:
Is it deliberate that here and further below, we're not using a separate variable for the primary button border? Secondary buttons have a separate variable.

Reply:
historical most likely, not intentional as far as I know

Reply:
Button variable API should be expanded IMO